build.gradle

```diff
@@ -0,0 +1,278 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java application project to get you started.
+ * For more details take a look at the 'Building Java & JVM projects' chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/7.4.2/userguide/building_java_projects.html
+ */
+
+plugins {
+	id 'java'
+	id 'war'
+	id 'jacoco'
+	id 'org.ajoberstar.grgit' version '4.1.1'
+}
+
+repositories {
+	mavenCentral()
+}
+// Warning - these values are hard coded in AuthController.java
+def JAR_TEMPLATE_DIR = 'kbase_auth2_templates'
+def TEMPLATE_LIST_FILE_NAME = "templates.manifest"
+
+task buildGitCommitFile {
+	doLast {
+		def commitId = grgit.head().id
+		// is there a varible for builddir/classe/java/main?
+		file("$buildDir/classes/java/main/us/kbase/auth2/gitcommit").text = commitId
+	}
+}
+
+compileJava {
+	if (JavaVersion.current() <= JavaVersion.VERSION_1_8) {
+		// TODO BUILD remove when we no longer support java 8
+		java.sourceCompatibility = JavaVersion.VERSION_1_8
+		java.targetCompatibility = JavaVersion.VERSION_1_8
+	} else {
+		options.release = 8
+	}
+	finalizedBy buildGitCommitFile
+}
+
+test {
+	/* 
+	 * TODO TEST Figure out why tests fail without this and remove. Might have something to do
+	 * with the stfuLoggers() call in many of the tests, might kill logging for tests that
+	 * require it
+	 * Although it seems to make Mongo start up correctly as well which is odd
```

Comment:
same issue in the workspace -- there seems to be some sort of interaction between gradle and the loggers that means tests only pass with `forkEvery` on